src/microbe_stage/Microbe.Interior.cs

```diff
@@ -378,6 +378,34 @@ public void SetInitialCompounds()
         {
             Compounds.AddCompound(entry.Key, entry.Value);
         }
+
+        var setting = GameWorld.WorldSettings;
+
+        // Fraction of day that is night
+        var nightTimeFraction = 1 - setting.DaytimeFraction;
+
+        // Shifts fraction so the day ends exactly 1 (ignoring padding).
+        var shiftedDay = (GameWorld.LightCycle.FractionOfDayElapsed + nightTimeFraction / 2 +
+            Constants.INITIAL_COMPOUND_EVENING_PADDING_FRACTION) % 1;
+
+        if (shiftedDay < nightTimeFraction)
+        {
+            // This should never trigger, but we need to assure the linter that
+            // organelles is never null here.
+            if (organelles == null)
+                throw new ArgumentNullException();
+
+            var remainingNight = nightTimeFraction - shiftedDay;
+            double remaningNightTime =
```

Comment:
Also just noticed as I opened this in my IDE that there's a typo here (`remaning`).